backend/src/main/java/com/soen341/instagram/exception/like/MultipleLikeException.java

```diff
@@ -8,7 +8,7 @@
 {
 	public MultipleLikeException()
 	{
-		super("The content can only be liked once by the user");
+		super("User can only like once.");
 	}
```

Comment:
I thought the previous message was clearer but this is fine too

Reply:
Agreed.